Week02/README.md

```diff
@@ -64,3 +61,9 @@
 - [Extra Long Factorials](https://www.hackerrank.com/challenges/extra-long-factorials?h_r=internal-search)
 	- [Bedir's Solution](https://github.com/BedirT/AlgorithmsL/blob/master/Problems/HackerRank/Algorithms/Implementation/Extra%20Long%20Factorial.cpp)
 	- [Nadide's Solution](https://github.com/nadide/ACM-ICPC/blob/master/problems/hackerrank/extraLongFactorials.c)
+- [GCD of Large Number](https://www.codechef.com/problems/GCD2)
+	- [Editorial](https://www.geeksforgeeks.org/gcd-of-two-numbers-when-one-of-them-can-be-very-large-2/)
+- [Modular GCD](https://www.codechef.com/AUG18B/problems/GCDMOD)
+- [Number of Balanced Binary Trees of Height h](https://www.geeksforgeeks.org/count-balanced-binary-trees-height-h/)
```

Comment:
Remove this one, it is not a clear question.

Reply:
Which question is to remove ?

Reply:
Number of Balanced Binary Trees of height h